catalog.yml

```diff
@@ -153,3 +153,7 @@
 - repo_url: https://github.com/greenelab/annorxiver_manuscript
   html_url: https://greenelab.github.io/annorxiver_manuscript/
   preprint_citation: doi:10.1101/2021.03.04.433874
+- repo_url: https://github.com/lotusnprod/rootstock
+  html_url: https://lotusnprod.github.io/rootstock/v/freeze
+  thumbnail_url: https://github.com/lotusnprod/rootstock/blob/7441faecc74dd02a103b098824329a1557fcd314/content/images/magicTree.svg
```

Comment:
A bit worried about the size of this SVG (9.11 MB) and potential complexity of rendering this for browsers.

My recommendation would be to create a PNG export of this image and name it `thumbnail.png`. Manubot will then detect this and you won't need to set `thumbnail_url` here. This thumbnail will also be applied when you share on social media. More info at https://github.com/manubot/rootstock/blob/main/USAGE.md#thumbnail

Reply:
I did upload a thumbnail.png, hope it is better now, thank you for the link!

Reply:
Yep for example at https://cards-dev.twitter.com/validator? I see

![image](https://user-images.githubusercontent.com/1117703/114111857-070b2d00-98a9-11eb-9d0b-e365cfa670ba.png)